foreman_salt.gemspec

```diff
@@ -15,4 +15,5 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'deface', '< 2.0'
   s.add_dependency 'foreman-tasks', '~> 0.8'
+  s.add_dependency 'foreman-remote-execution', '~> 1.8.0'
```

Comment:
I guess the gem is called foreman_remtoe_execution (underscore instead of dashes).